app/workers/regeneration_worker.rb

```diff
@@ -3,7 +3,7 @@
 class RegenerationWorker
   include Sidekiq::Worker
 
-  sidekiq_options queue: 'pull', backtrace: true, unique: :until_executed
```

Comment:
No more backtraces?

Reply:
This class does not throw exceptions, it's not necessary. If it ever does throw one, it'll still show up in the logs, just without a huge backtrace.